lapce-ui/src/editor/view.rs

```diff
@@ -862,6 +897,16 @@ impl Widget<LapceTabData> for LapceEditorView {
             }
         }
 
+        if data.config.editor.autosave_interval > 0
+            && *data.focus == self.view_id
```

Comment:
Shouldn't the timer be triggered when the document is changed rather than focus changed? 

So

 ```
editor_data.doc.rev() != old_editor_data.doc.rev()
```

Reply:
done